### ISSUE TITLE
About checkpoint

### ISSUE THREAD

**Issue by Fly-Yun-Maker** [yassouali/CCT#61]
I looked carefully at your code and found that the model does not save the learning rate, will it have any effect if the training is aborted and the previous round is continued?

**Comment by yassouali** [comment]
Hi @NJNUCS 

Sorry for the delayed response

Do you mean the LR used for fine-tuning or the initial lr?
For the initial value, you can find the LR in the config file under "optimizer", note that the LR of the backbone is 1/10 of this lr value since the backbone is pretrained (see this line [here](https://github.com/yassouali/CCT/blob/cf98ea7e6aefa7091e6c375a9025ba1e0f6e53ca/base/base_trainer.py#L36)). So if you want to fine-tune, I would set all learning rates to 1/10 of this value.
